### PR TITLE
chore: upgrade jest everywhere

### PR DESCRIPTION
**Pull request by riqwan**
not much to do in a long train journey 😄

### CODE REVIEW

**Review by carlos-r-l-rodrigues — APPROVED** [review]

**Comment by olivermrbl** [comment]
What's new in the upgrade? :)

**Comment by riqwan** [comment]
> What's new in the upgrade? :)

@olivermrbl Built in sharding support. I'm sure there's more, but I stopped reading after this 😄

File: packages/core/medusa-test-utils/src/medusa-test-runner.ts [hunk 1]
```diff
@@ -87,7 +87,9 @@ export function medusaIntegrationTestRunner({
   testSuite: <TService = unknown>(options: MedusaSuiteOptions<TService>) => void
 }) {
   const tempName = parseInt(process.env.JEST_WORKER_ID || "1")
+  console.log("tempName -- ", tempName)
```
Comment (sradevski) [inline packages/core/medusa-test-utils/src/medusa-test-runner.ts]: Is this intentional?

**Review by sradevski — APPROVED** [review]
Nice! I am surprised nothing broke? 😄

File: integration-tests/http/jest.config.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 process.chdir(__dirname)
 
 module.exports = {
-  name: "HTTP Integration Tests",
```
Comment (adrien2p) [inline integration-tests/http/jest.config.js:4]: Q: why dont we need that anymore? what was it used for?

File: packages/core/medusa-test-utils/src/medusa-test-runner.ts [hunk 3]
```diff
@@ -96,7 +98,7 @@ export function medusaIntegrationTestRunner({
     schema,
     debug,
   }
-
+  console.log("dbConfig -- ", dbConfig)
```
Comment (adrien2p) [inline packages/core/medusa-test-utils/src/medusa-test-runner.ts]: suggestion: rm

File: packages/medusa/src/loaders/__tests__/feature-flags.spec.ts [hunk 4]
```diff
@@ -77,11 +77,11 @@ describe("feature flags", () => {
       join(filesystem.basePath, "flags")
     )
 
-    expect(flags.isFeatureEnabled("flag_1")).toEqual(true)
+    expect(flags.isFeatureEnabled("flag_1")).toEqual(false)
```
Comment (adrien2p) [inline packages/medusa/src/loaders/__tests__/feature-flags.spec.ts:80]: Q: how come all the expectation are inverted? 😂

Comment (riqwan) [reply packages/medusa/src/loaders/__tests__/feature-flags.spec.ts:80]: Wanted to test all cases, check line 100, that test should've failed. But it passed for so long :D

File: integration-tests/http/jest.config.js (hunk 2 above)
Comment (riqwan) [reply integration-tests/http/jest.config.js:4]: didn't see it being used anywhere and the new jest config doesn't support it

**Comment by olivermrbl** [comment]
> Built in sharding support. I'm sure there's more, but I stopped reading after this

@riqwan, nice haha! 😄

**Review by olivermrbl — APPROVED** [review]

**Pull request merged**